src/components/Table/TableHead.jsx

```diff
@@ -0,0 +1,17 @@
+import PropTypes from 'prop-types';
+
+export function TableHead({ titles }) {
+  return (
+    <thead>
+      <tr>
+        {Object.values(titles).map((item) => (
```

Comment:
Can we use just already prepared data ? How to understand that we need to pass object but not an array ? Please also use more generic names like `items`, `elements`, because its's more clear what's expected as an input. Apply the same changes in other component with the same structure.

Reply:
```
export function TableHead({ items }) {
  return (
    <thead>
      <tr>
        {items.map((item) => (
          <th key={item}>{item}</th>
        ))}
      </tr>
    </thead>
  );
}
```